FreeRTOS/FreeRTOS
# Add SMP disable multiple priorities on target test

Add SMP disable multiple priorities on target test

Description
-----------
Adding the on target test to verify that
> The user shall be able to configure the scheduler to not run a lower priority task and a higher priority task simultaneously.


Test Steps
-----------
Test result
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/disable_multiple_priorities/disable_multiple_priorities.c:229:Test_DisableMultiplePriorities:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
OK
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
